transport/wsn/src/test/java/org/kaazing/gateway/transport/wsn/specification/ws/acceptor/BaseFramingIT.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package org.kaazing.gateway.transport.wsn.specification.ws;
+package org.kaazing.gateway.transport.wsn.specification.ws.acceptor;
 
 import static org.kaazing.test.util.ITUtil.createRuleChain;
```

Comment:
ITUtil.createRuleChain is something I added because there was a lot of confusion about ordering of rules in the rule chain in our various robot tests and I wanted to standardize it so without a duplicate it everywhere. We had been getting test failures due to bad ordering. Also the method automatically and see useful rule to get diagnostic trace when tests fail, very helpful for diagnosing build failures on Travis. Admittedly it may not be necessary to use this in the specification projects. It was mainly aimed at robot tests which are running Gateway.